### PR TITLE
chore(locales): EN UI update

### PR DESCRIPTION
**Pull request by Sherlock113**
Signed-off-by: sherlock <sherlockxu@yunify.com>

**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

Update En UI

### CODE REVIEW

**Comment by Sherlock113** [comment]
/assign @leoendless
Thank you.

File: src/locales/en/app.js [hunk 1]
```diff
@@ -21,17 +21,16 @@ export default {
 
   APP_TEMPLATE_DESC:
     'KubeSphere provides lifecycle management of app templates. You can upload or create new app templates and test them quickly. In addition, you can publish your app templates to App Store so that other users can purchase and deploy them.',
-
   APP_TEMPLATE_CREATE_DESC:
     'KubeSphere provides lifecycle management of app templates. You can upload or create new app templates and test them quickly. In addition, you can publish your app templates to App Store so that other users can purchase and deploy them.',
+  APP_TEMPLATES_DESC:
+    'KubeSphere provides application full lifecycle management. You can upload or create new application templates and test them quickly. In addition, you can publish your application to App Store so that other users deploy it in one-click wat.',
```
Comment (rayzhou2017) [inline src/locales/en/app.js:27]: in one-click wat, here `wat` is a typo?

Comment (Sherlock113) [reply src/locales/en/app.js:27]: Yes, I will change it and submit again.

Comment (leoendless) [reply src/locales/en/app.js]: Resolved ？

Comment (Sherlock113) [reply src/locales/en/app.js]: Yes

**Pull request merged**